decidim-core/db/migrate/20180724103814_add_content_blocks.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddContentBlocks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :decidim_content_blocks do |t|
+      t.integer :decidim_organization_id, null: false, index: true
+      t.string :manifest_name, null: false, index: true
+      t.string :scope, null: false, index: true
+      t.jsonb :options
+      t.datetime :published_at
```

Comment:
Add an index here (I guess we're only showing the published ones)

Reply:
Maybe we need a combined index with `decidim_organization_id`, can you try an run an `explain` on the query to see which is better?